apps/form-system/web/src/components/Screen/components/ExternalData/ExternalData.tsx

```
@@ -0,0 +1,82 @@
+            <Text variant="h4" color="blue400">
+              {formatMessage(webMessages.myPagesTitle)}
+            </Text>
+            <Text>
+              <Markdown>
+                {formatMessage(webMessages.myPagesDescription)}
+              </Markdown>
+            </Text>
+          </div>
+
+          {certificationTypes?.map((certificationType) => (
+            <div>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)